lib/ood_core/job/adapters/kubernetes/batch.rb

```diff
@@ -25,6 +26,7 @@ def initialize(options = {})
     @all_namespaces = options.fetch(:all_namespaces, false)
     @username_prefix = options.fetch(:username_prefix, '')
     @namespace_prefix = options.fetch(:namespace_prefix, '')
+    @enable_supplemental_groups = options.fetch(:enable_supplemental_groups, false)
```

Comment:
OK - last thing. Is this the right name?  Sounds like it's disabling it altogether. Shouldn't it be `auto_populate_groups` or `auto_supplemental_groups`? I mean I think you can infer supplemental groups from the word _groups_, but you never know what people may assume.

Reply:
I renamed to `auto_supplemental_groups` per suggestion.